manual.md

```diff
@@ -67,69 +65,72 @@ You can run the GUI from the command line, or by double-clicking it in a folder.
 You should see a window similar to this:<br/>
 <img src="https://sciinstitute.github.io/cleaver.pages/images/application.png"><br/>
 Load the spheres in <code>src/test/test_data/input</code>
-either with <code>ctrl+v</code> or <code>File -> Load Volume</code>,
+either with <code>ctrl+v</code> or <code>File -> Import Volume</code>,
        or load your own indicator functions or segmentation file. <br/>
-       *Dialog Indicator Function Check* Click the check in the dialog if you are
+       *Dialog Indicator Function Check:* Click the check in the dialog if you are
        importing individual indicator functions.<br/>
-       *Blending Function Sigma* Choose a sigma for pre-process smoothing either
+       *Blending Function Sigma:* Choose a sigma for pre-process smoothing either
        your segmentation labels or indicator functions to avoid stair-step aliasing.<br/>
        **Sizing Field Creator**<br/>
        This tool allows a user to set parameters for the cleaving sizing field.<br/>
-       *Scale Factor* This is a tool to alter the sampling size of a volume.
-       Smaller sampling creates coarser meshes faster.<br/>
-       *Size Multiplier* This is the multiplier for the sizing field creation.<br/>
-       *Lipschitz* This is the grading of the sizing field. <br/>
-       *Padding* Added a volume buffer around the data. This is useful when volumes intersect near the boundary.<br/>
-       *Element Sizing Method* Select whether to adaptively resize tets for more detail at volume interactions, or
-       to keep tet sizes constant based on the sample scale.<br/>
-       *Compute Sizing Field* Once you have your desired parameters, click this to create the sizing field.
-       This is assuming a volume has been loaded (ctrl+v or File->Import Volume). New information will be added
+       *Sampling Rate:* the sampling rate of the input indicator functions or calculated indicator functions 
+       from segmentation files. The default sample rate will be the dimensions of the volume. Smaller sampling 
+       creates coarser meshes faster.<br/>
```

Comment:
creates coarser meshes.  Adjusting this parameter will also affect Cleaver's runtime, with smaller values running faster.